pkg/util/oom/oom_linux.go

```diff
@@ -78,12 +75,22 @@ func applyOOMScoreAdj(pid int, oomScoreAdj int) error {
 	value := strconv.Itoa(oomScoreAdj)
 	var err error
 	for i := 0; i < maxTries; i++ {
-		if err = ioutil.WriteFile(oomScoreAdjPath, []byte(value), 0700); err != nil {
+		f, err := os.Open(oomScoreAdjPath)
+		if err != nil {
+			if syscallNotExists(err) {
+				return os.ErrNotExist
+			}
+			err = fmt.Errorf("failed to apply oom-score-adj to pid %d (%v)", pid, err)
+			continue
+		}
+		if _, err := f.Write([]byte(value)); err != nil {
 			if syscallNotExists(err) {
```

Comment:
Is it possible for the Open() to succeed but the Write to fail with a "file not found" error?